### ISSUE TITLE
[FEATURE] Protect application access with password

### ISSUE THREAD

**Issue by gpor0** [mendhak/gpslogger#840]
We would like to secure unintended application access with password so only administrator can manage application settings & turn logging on or off. Logging should start at device boot and acquire location in background and should not be stopped.

**Comment by ranjeewainfo** [comment]
Yes this is a good idea. So it will keep settings safe. Also it helps protect saved information such as usernam password access by unauthorized persons.
